app/src/main/java/org/mozilla/fenix/experiments/NimbusSetup.kt

```diff
@@ -96,3 +100,17 @@ fun createNimbus(context: Context, url: String?): NimbusApi {
         NimbusDisabled()
     }
 }
+
+/**
+ * Classifies which errors we should forward to our crash reporter or not. We want to filter out the
+ * non-reportable ones if we know there is no reasonable action that we can perform.
+ *
+ * This fix should be upstreamed as part of: https://github.com/mozilla/application-services/issues/4333
```

Comment:
Do we need a follow up issue to remove this comment when we do upstream this fix?

Reply:
Let's wait to see what the outcome of the discussion is in that ticket. There could be alternative solutions to this problem.